src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -306,6 +322,83 @@
   </div>
 {% endmacro %}
 
+{% macro mediationSuccessForDefendantClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('You both agreed a settlement through mediation') }}</h2>
+    <p>{{ t('The claimant can’t request a County Court Judgment against you unless you break the terms') }}</p>
+    <p>{{ internalLink('Contact ' + claim.claimData.claimant.name, DashboardPaths.contactThemPage.evaluateUri({ externalId: claim.externalId })) }}
+      {{ t('if you need their payment details. Make sure you get receipts for any payments.') }}</p>
+  </div>
+{% endmacro %}
+
+{% macro mediationSuccessForClaimantClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('You both agreed a settlement through mediation') }}</h2>
+  </div>
+{% endmacro %}
+
+{% macro mediationFailedForOnlineDQClaimantClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('Mediation was unsuccessful') }}</h2>
+    <p>{{ t('You weren’t able to resolve your claim against {{defendantName}} using mediation.',{ defendantName: claim.claimData.defendant.name }) }}</p>
+    {% if claim.pilotCourt === YesNoOption.YES %}
+      <p>{{ t('The court will review the case. We’ll contact you to tell you what to do next.') }}</p>
+    {% else %}
+      <p>{{ t('You’ll have to go to a hearing. We’ll contact you with the details.') }}</p>
+    {% endif %}
+  </div>
+{% endmacro %}
+
+{% macro mediationFailedForOfflineDQClaimantClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('Mediation was unsuccessful') }}</h2>
+    <p>{{ t('You weren’t able to resolve your claim against {{ defendantName }} using mediation.',{ defendantName: claim.claimData.defendant.name }) }}</p>
+    <p>{{ t('You’ll have to go to a hearing. We’ll contact you with the details.') }}</p>
+    <p>{{ t('You need to {{ completeDQ }} form to give us more information in case there’s a hearing.',
```

Comment:
this content is wrong. it should be `You need to complete a directions questionnaire form to give us more information for the hearing.`